### PR TITLE
Feature/new site

### PR DESCRIPTION
**Pull request by katycaballeros**
Hey Em! I made updates to the project. Check it out!

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <a href="about.html">About</a>
+    <section>
+      <h2>Chihuahua</h2>
+      <img src="images/sunbathing-chihuahua.jpg" height="500px" />
+      <ul>
+        <li>World's smallest dog is a chuihuahua</li>
+        <ul><li>weighs less than one pound</li></ul>
+        <li>Has two head shapes</li>
+        <ul><li>apple and deer shape</li></ul>
+        <li>Katy will get one for Saint Patrick's Day</li>
```
Comment (emzinnia) [inline Index.html:23]: Hmm...wasn't this supposed to be your birthday?

File: about.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!Doctype html>
+<html>
+  <head>
+    <title>About Cute Doggos</title>
```
Comment (emzinnia) [inline about.html:4]: Maybe we could add the new font here as well?

**Review by emzinnia — CHANGES_REQUESTED** [review]
Looking great! Only thing I think you should do is add the new font to the about page.

File: Index.html (hunk 1 above)
Comment (katycaballeros) [reply Index.html:23]: Saint Patrick's day is...my birthday...

File: about.html (hunk 2 above)
Comment (katycaballeros) [reply about.html:4]: Isn't it tacky to have different fonts across a website?

Comment (emzinnia) [reply about.html:4]: Sometimes. But I meant the same font *everywhere*

Comment (katycaballeros) [reply about.html:4]: ahhhhhh yes so we need to change the about page to Muli

**Review by emzinnia — APPROVED** [review]

**Pull request merged**